help/md/dnFastBirthDeathShift.md

```diff
@@ -0,0 +1,86 @@
+## name
+dnFastBirthDeathShift
+## title
+The relaxed birth-death-shift process
+## description
+Simulates a tree under the four-parameter birth-death-shift process 
+## details
+This distribution represents the birth-death-shift process, which has four
+possible events:
+
+- a speciation event (lambda)
+- an extinction event (mu)
+- a speciation rate shift event (alpha)
+- an extinction rate shift event (beta)
+
+The possible speciation rates (lambda) and extinction rates (mu) are
+assumed to be distributed according to discretized base distributions.
+Currently, the log-normal base distribution is hard coded. The log-normal
+base distributions have location parameters (speciationScale, extinctionScale) 
+and spread parameters (speciationSD, extinctionSD), which can be fixed or
+estimated using prior distributions.
+
+In comparison with dnCDBDP, the dnFastBirthDeathShift distribution is 
+simplified and more optimized, allowing it to be significantly faster than 
+the implementation in dnCDBDP (some tests indicate 60-70x performance).
```

Comment:
This is super nitpicky, but you could perhaps highlight argument and distribution names using backticks (` `` `).